src/config.js

```diff
@@ -322,6 +326,11 @@ function describeApi (rootDir) {
         language: LANGUAGE_JS_LATEST,
         requiredApi: jsDescribeApi({ rootDir })
       };
+    } else if (existsSync(path.join(rootDir, 'api.py'))) {
```

Comment:
Yes, currently.
There are a few things we need to figure out before allowing multiple runtimes at once. After we've merged this branch into master I'll open a PR to discuss :)